client/packages/invoices/src/Prescriptions/LineEditView/PrescriptionLineEditForm.tsx

```diff
@@ -569,14 +585,30 @@ const summarise = (
   // Summarise counts in words
   const summary: string[] = [];
   Object.entries(counts).forEach(([size, { unitName, count }]) => {
-    const unitWord = t('label.unit-plural', {
-      count,
-      unit: unitName,
-    });
-    if (Number(size) > 1) {
-      const packs = NumUtils.round(count / Number(size), 3);
-      summary.push(t('label.packs-of-size', { packs, count, size, unitWord }));
+    const packSize = Number(size);
+    if (packSize > 1) {
+      const packs = NumUtils.round(count / packSize, 3);
+      const packWord = pluralize(t('pack'), packs);
+      const unit = pluralize(t('unit'), count);
+      const unitWord = pluralize(
+        t('label.unit-type', { unit: unitName }),
+        packSize
+      );
```

Comment:
Again, don't need to pluralize, just use existing pluralisation rules for "unit" key.